packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -196,94 +200,124 @@ function ComboBox(props) {
           </span>
         </button>
         {state.isOpen &&
-          <ListBoxPopup
-            {...listBoxProps}
-            // Use virtual focus to get aria-activedescendant tracking and
-            // ensure focus doesn't leave the input field
-            shouldUseVirtualFocus
-            listBoxRef={listBoxRef}
-            popoverRef={popoverRef}
-            state={state} />
+          <Popover popoverRef={popoverRef} isOpen={state.isOpen} onClose={state.close}>
+            <ListBox
+              {...listBoxProps}
+              listBoxRef={listBoxRef}
+              state={state} />
+          </Popover>
         }
       </div>
     </div>
   );
 }
 
-function ListBoxPopup(props) {
+<ComboBox label="Favorite Animal">
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+
+### Popover
+
+The `Popover` component is used to contain the popup listbox for the ComboBox.
+It can be shared between many other components, including [Select](useSelect.html),
+[Menu](useMenuTrigger.html), [Dialog](useOverlayTrigger.html), and others.
+See [useOverlayTrigger](useOverlayTrigger.html) for more examples of popovers.
```

Comment:
Do we need this line? We already link to useOverlayTrigger via the Dialog link, and the useOverlayTrigger doc only has a popover example (unless we are trying to highlight that doc so users understand they can use that hook and pass in their desired `type` as a prop to the useOverlayTrigger hook?)